requirements.txt

```diff
@@ -3,6 +3,7 @@ selenium==3.141.0
 webdriver_manager==3.3.0
 requests==2.21.0
 urllib3==1.24.2
+polling2
```

Comment:
Why do we need new dependency when we have built-in polling?

import polling

Reply:
there is no built-in polling

Reply:
Sorry, was confused by my environment